src/pages/CustomerRequisitionPage.js

```diff
@@ -117,9 +137,94 @@ export const CustomerRequisition = ({
         sortKey={sortKey}
       />
     ),
-    [sortKey, isAscending]
+    [columns, sortKey, isAscending]
+  );
+
+  const ItemIndicatorToggles = useMemo(
+    () => [
+      {
+        text: programStrings.items,
+        isOn: !showIndicators,
+        onPress: onHideIndicators,
+      },
+      {
+        text: programStrings.indicators,
+        isOn: showIndicators,
+        onPress: onShowIndicators,
+      },
+    ],
+    [showIndicators]
+  );
+
+  const ItemIndicatorToggle = () => <ToggleBar toggles={ItemIndicatorToggles} />;
+
+  const ButtonSetSuppliedToRequested = useCallback(
+    () => (
+      <PageButton
+        style={globalStyles.topButton}
+        text={buttonStrings.use_requested_quantities}
+        onPress={onSetSuppliedToRequested}
+        isDisabled={isFinalised}
+      />
+    ),
+    [isFinalised]
+  );
+
+  const ButtonSetSuppliedToSuggested = useCallback(
+    () => (
+      <PageButton
+        style={globalStyles.topButton}
+        text={buttonStrings.use_suggested_quantities}
+        onPress={onSetSuppliedToSuggested}
+        isDisabled={isFinalised}
+      />
+    ),
+    [isFinalised]
+  );
+
+  const indicatorCodes = useMemo(() => indicators.map(indicator => indicator.code), [indicators]);
+  const selectedIndicatorCode = useMemo(() => selectedIndicator.code, [selectedIndicator]);
```

Comment:
bit nicer to just use selectors for these

Reply:
Much nicer. e4bf808462abf39d9bf95ef8c410b6a8aee28a92.